### ISSUE TITLE
Unable to add as an accessory to your TV 'HDMI' input

### ISSUE THREAD

**Issue by msbc42** [AntonioMeireles/homebridge-vieramatic#78]
Following log messages at startup:
```

[5/7/2021, 9:29:22 AM] [PanasonicVieraTV] Unable to add as an accessory to your TV 'HDMI' input:
{
  "id": "1",
  "name": "ATV"
}

If you do believe that your homebridge's 'config.json' is in order and has absolutelly no duplicated entries for HDMI inputs then please fill a bug at https://github.com/AntonioMeireles/homebridge-vieramatic/issues, otherwise just remove or fix the duplicated stuff.
[5/7/2021, 9:29:22 AM] [PanasonicVieraTV] successfully loaded Main TV

```
HomeBridge: 1.3.4
Plug-in: 3.1.13
Config
```

            "tvs": [
                {
                    "friendlyName": "Main TV",
                    "ipAddress": "X.X.X.X",
                    "encKey": "*******************",
                    "appId": "*****************",
                    "hdmiInputs": [
                        {
                            "id": "1",
                            "name": "ATV"
                        },
                        {
                            "id": "2",
                            "name": "Sonos"
                        },
                        {
                            "id": "3",
                            "name": "Fetch"
                        },
                        {
                            "id": "4",
                            "name": "Blu-ray"
                        }
                    ],
                    "disabledAppSupport": false,
                    "customVolumeSlider": false
                }
            ],
            "platform": "PanasonicVieraTV"
```

**Comment by AntonioMeireles** [comment]
@msbc42 

strange. coyld you please attach the contents of your `accessories/vieramatic.json` file. (the path is reative to where the directory where homebridge's `config.json` is located.

Thanks in advance.

**Comment by AntonioMeireles** [comment]
@msbc42 

attach here also please the full logs of homebridge booting.

**Comment by msbc42** [comment]
Boot log:
[homebridge.log](https://github.com/AntonioMeireles/homebridge-vieramatic/files/6445018/homebridge.log)

Config JSON compressed:
[vieramatic.json.zip](https://github.com/AntonioMeireles/homebridge-vieramatic/files/6445019/vieramatic.json.zip)

**Comment by AntonioMeireles** [comment]
@msbc42 

Thanks!  at sight your `homebridge.log` looks incomplete. the error isn't there.. could you please check and resend ?

thanks in advancde and a great weekend!

**Comment by msbc42** [comment]
The "Unable to add as an accessory to your TV 'HDMI' input" message is on line 68. Are you expecting something else/more?

**Comment by AntonioMeireles** [comment]
@msbc42  

sorry - my fault. was looking at another `homebridge.log` in my `~/Downloads/` folder ... could you please delete the existing `vieramatic.json`, then restart homebridge and see if the issue persists ?

thanks for your patience!

**Comment by msbc42** [comment]
Deleting the vieramatic.json seems to have fixed the error.

Thanks

**Comment by AntonioMeireles** [comment]
@msbc42 

this was due to a **real** bug in my side after all.  i wasn't taking in account that people may need to rename (previously configured) HDMI interfaces ...

 fixed in just published `homebridge-vieramatic@3.1.14`
 
 thanks once again for your patience!
